### ISSUE TITLE
RUNSAMPLEOPTIONS not working on version 1.4.0?

### ISSUE THREAD

**Issue by mmelendrez** [VDBWRAIR/ngs_mapper#223]
```
[melanie.melendrez@amedwrair015380 SubjCHD10_034_044]$ ll
total 32
-rw-r--r--. 1 melanie.melendrez domain users  5296 Mar 28 13:15 config_quasi.yaml
-rw-r--r--. 1 melanie.melendrez domain users  5295 Mar 28 13:09 config.yaml
-rwxr-xr-x. 1 melanie.melendrez domain users 10681 Mar 28 12:40 KDH0053A_Thailand_NC_2010.fasta
-rwxr-xr-x. 1 melanie.melendrez domain users   114 Mar 28 12:40 samplesheet_SubjCHD10_034_044
[melanie.melendrez@amedwrair015380 SubjCHD10_034_044]$ RUNSAMPLEOPTIONS"-c config_quasi.yaml" runsamplesheet.sh /media/VD_Research/NGSData/ReadsBySample/ samplesheet_SubjCHD10_034_44
bash: RUNSAMPLEOPTIONS-c config_quasi.yaml: command not found
```
Am I implementing this incorrectly? I am implementing per the latest docs.

**Comment by mmelendrez** [comment]
Tried with a single sample and that appears to be working:
```
[melanie.melendrez@amedwrair015380 SubjCHD10_034_044]$ runsample -od Day1_10672 -c config_quasi.yaml /media/VD_Research/NGSData/ReadsBySample/10672/ KDH0053A_Thailand_NC_2010.fasta Day1_10672
2016-03-28 13:28:02,892 -- INFO -- runsample       --- Starting Day1_10672 --- 
2016-03-28 13:28:02,892 -- INFO -- runsample       --- Using custom config from config_quasi.yaml ---
```

**Comment by averagehat** [comment]
It looks like you're missing an `=` after `RUNSAMPLEOPTIONS`

```bash
RUNSAMPLEOPTIONS"-c config_quasi.yaml" runsamplesheet.sh /media/VD_Research/NGSData/ReadsBySample/ samplesheet_SubjCHD10_034_44
```
Should be 
`RUNSAMPLEOPTIONS="-c config_quasi.yaml" runsamplesheet.sh /media/VD_Research/NGSData/ReadsBySample/ samplesheet_SubjCHD10_034_44`

**Comment by mmelendrez** [comment]
OK - this needs to be fixed at http://ngs-mapper.readthedocs.org/en/latest/config.html

the command there is:
```
$> RUNSAMPLEOPTIONS"-c config.yaml" runsamplesheet.sh /path/to/NGSData/ReadsBySample samplesheet.tsv
```

**Comment by averagehat** [comment]
Indeed. Fixed in 0f85f47
